Update .travis.yml

bluetimer opened the pull request:
added travis declaration 
Closes https://github.com/manu-chroma/username-availability-checker/issues/38

andrewda commented on .travis.yml:
```diff
@@ -1,6 +1,8 @@
-  - 3.6
+- "3.4"
+- "3.5"
+- "3.6"
```
Indent these lines with 2 spaces. Also quotes aren't necessary.

andrewda commented:
Also, in the future, please try to keep working on the same PR instead of closing it and creating a new one https://github.com/manu-chroma/username-availability-checker/pull/119

yukiisbored commented on .travis.yml:
```diff
@@ -1,7 +1,9 @@
+  - 3.5  
   - 3.6
-
+    
```
Remove this unwanted whitespaces. Try keep your amount of changes as small as you can.

yukiisbored commented:
Also, squash your commits since this is just one change. Feel free to ask us on how to do it in the Zulip/Gitter chat.

yukiisbored commented:
Your commit message does not follow our commit guidelines. See https://coala.io/commit.

andrewda commented on .travis.yml:
```diff
@@ -1,7 +1,9 @@
 language: python
 python:
+  - 3.4                                            
```
Also remove trailing whitespace on this line and on the line below

bluetimer commented:
.travis.yml:added python versions in travis declaration  
added python 3.4 3.5 3.6 version in travis 
closes https://github.com/manu-chroma/username-availability-checker/issues/38

andrewda commented on README.md:
```diff
@@ -48,6 +48,8 @@ To use, go to address generated by flask app in a browser.
 
+Runs on python 3.4,3.5
+
 Tested on python 3.6
```
Maybe instead of the addition above, something like,

"Tested on Python 3.4, 3.5 and 3.6"
